frontend/pages/SoftwarePage/SoftwareTitleDetailsPage/SoftwareInstallerCard/InstallerDetailsWidget/_styles.scss

```
@@ -6,7 +6,11 @@
     font-size: $x-small;
     font-weight: $bold;
     @include ellipse-text;
-    max-width: 48vw;
+    max-width: 60vw;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)